requirements/pip.txt

```diff
@@ -0,0 +1,11 @@
+#
+# This file is autogenerated by pip-compile with python 3.8
+# To update, run:
+#
+#    make upgrade
+#
+wheel==0.37.1             # via -r requirements/pip.in
+
+# The following packages are considered to be unsafe in a requirements file:
+pip==22.1.2               # via -r requirements/pip.in
+setuptools==62.4.0        # via -r requirements/pip.in
```

Comment:
`constraints.txt` needs to include `common_constraints.txt` in it to follow common constraints correctly.